services/platform/app/features/automations/triggers/components/secret-reveal-dialog.tsx

```
@@ -0,0 +1,104 @@
+          className="bg-amber-50 dark:bg-amber-950/20 border border-amber-200 dark:border-amber-800 rounded-lg p-3"
+          role="alert"
+        >
+          <p className="text-sm text-amber-800 dark:text-amber-200">
+            {warning}
+          </p>
+        </div>
+        {secrets.map((secret, index) => (
+          <Stack key={secret.label} gap={1}>
+            <label className="text-sm font-medium text-foreground">
+              {secret.label}
+            </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)